src/giftCards/GiftCardsList/GiftCardListSearchAndFilters/GiftCardListSearchAndFilters.tsx

```diff
@@ -39,14 +40,12 @@ import {
 const GiftCardListSearchAndFilters: React.FC = () => {
   const navigate = useNavigator();
   const intl = useIntl();
+  const [selectedChannel] = useLocalStorage("channel", "");
```

Comment:
**question:** Is this local setting? If so, it might be better to use more descriptive key, e.g. `giftCardChannel`. If this is global then it's okay as it is.

Reply:
it is global - currently selected channel from home page 😄